example/lib/controls.js

```diff
@@ -0,0 +1,51 @@
+const allKeys = Array.prototype.slice.call(
```

Comment:
Simpler this way: `Array.from(document.getElementsByClassName(...), element => element.getElementsByTagName(...)[0].id)`